main_test.go

```diff
@@ -457,6 +457,22 @@ func newSingleContainerPodStatusList(podID, containerID string, podState, contai
 	}
 }
 
+func newContainerListWithAnnotations(podID, containerID string, podState, containerState vc.State, annotations map[string]string) []vc.PodStatus {
```

Comment:
You should not define a new function. Instead you should improve `newSingleContainerPodStatusList()` so that you can provide a map of annotations.

Reply:
I think its ok to have another function, we shouldnt be passing annotations when there is no need for it.

Reply:
We don't have to pass any annotations if we don't want to, we can just give an empty/nil map.

Reply:
I agree with @sboeuf - this is identical to `newSingleContainerPodStatusList()` except for the inclusion of the `Annotations`.

Reply:
Just a thought - you could always have `newSingleContainerPodStatusList()` call down to `newSingleContainerPodStatusListWithAnnotations()`, passing in the null annotation args etc. - so, have the no-annotations func as a thin shim/wrapper around the fuller func.

Reply:
@sboeuf I have removed the new function and added annotations parameter to the existing function.